bika/lims/browser/aggregatedanalyses/aggregatedanalyses.py

```diff
@@ -99,6 +120,24 @@ def __init__(self, context, request, **kwargs):
                          'state_title',
                          ]
              },
+            {'id': 'verified',
+             'title': _('Verified'),
+             'transitions': [],
+             'contentFilter': {
+                 'review_state': ['verified'],
+                 'cancellation_state': 'active', },
+             'columns': ['AnalysisRequest',
+                         'Worksheet',
+                         'Service',
+                         'Result',
+                         'Uncertainty',
+                         'Partition',
+                         'Method',
+                         'Instrument',
+                         'Analyst',
+                         'state_title',
+                         ]
```

Comment:
Please add `DetectionLimit` and `Specifications` columns at least, in same order as in `AnalysesView`